moveit_planners/trajopt/trajopt/CMakeLists.txt

```
@@ -0,0 +1,62 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(trajopt)
+
+add_compile_options(-std=c++11 -Wall -Wextra)
+
+find_package(catkin REQUIRED COMPONENTS
+  trajopt_sco
+  trajopt_utils
+  roscpp
+)
```

```suggestion
find_package(catkin REQUIRED COMPONENTS
  moveit_core
  trajopt_sco
  trajopt_utils
  roscpp
)
```

Shouldn't we have a dependency on `moveit_core` here? I got errors while compiling without it.